subsys/bluetooth/mesh/shell.c

```diff
@@ -2680,6 +2720,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(mesh_cmds,
 		      cmd_app_key_del, 3, 0),
 	SHELL_CMD_ARG(app-key-get, NULL, "<NetKeyIndex>", cmd_app_key_get, 2,
 		      0),
+	SHELL_CMD_ARG(transmit-param, NULL, "[<count: 0-7>"
```

Comment:
Command should probably be `net-transmit-get`. Fits more with the existing commands.